notebooks/1.1_Data_Ingestion-Intake.ipynb

```diff
@@ -27,9 +27,7 @@
    "source": [
```

Comment:
<p>The result I'm getting from the previous code cell is the following:</p><p> </p><p><code></code><code></p><p> named datasets: ['LC09_L2SP_105013_20240301_02_T1', 'LC09_L2SP_105014_20240301_02_T1', 'LC09_L2SP_105015_20240301_02_T1', 'LC09_L2SP_105016_20240301_02_T2', 'LC09_L2SP_105017_20240301_02_T2']</p><p></code><code></code></p><p> </p><p>So this code cell errors because it doesn't find "LC09_L2SP_094061_20240201_02_T2"</p><p> </p><p>Maybe instead we can just use something like this for this and the next code cell:</p><p> </p><p><code></code><code></p><p>item_key <strong>=</strong> list(cat.entries.keys())[0]</p><p>subcat <strong>=</strong> cat[item_key].read()</p><p>subcat</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/landsat-ml-cookbook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>